release-notes/6.0/install-linux.md

```diff
@@ -0,0 +1,57 @@
+# .NET 6 installation instructions for Linux
+
+The following instructions demonstrate installing .NET 6 on Linux. These instructions augment the more general [.NET install instructions](install.md), including installing with `.tar.gz` files, that work on multiple operating systems.
+
+[Linux package dependencies](linux-packages.md) describes the set of packages required to run .NET on Linux.
+
+## Distributions
+
+The following distributions are available for Linux:
+
+- .NET SDK: includes tools for building and testing applications, and includes the runtime distributions that follow.
+- .NET Runtime: includes the .NET runtime and libraries, enabling running console applications.
+- ASP.NET Core Runtime: includes the .NET and ASP.NET Core runtimes, enabling running console, and web applications.
+
+You are recommended to install the .NET SDK to develop and build applications, and to install one of the runtimes packages (like ASP.NET Core) to exclusively run applications.
+
+## Downloads
+
+Each of the distributions can be downloaded from:
+
+- [Microsoft .NET website](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [.NET 6 release notes](README.md)
+
+[Container images](https://hub.docker.com/r/microsoft/dotnet/) are provided for Linux (Alpine, Debian, and Ubuntu).
+
+## Install using deb/rpm packages
+
+Preview release installers are not available from the Microsoft package repositories. The steps below provide an easy way to install .NET 6 using your Distro package manager.
+
+**Note:** `curl` must be available on the system before running the following steps. Once you have confirmed that `curl` is available, complete the steps to download and install the latest .NET 6 Preview SDK and Runtime.
+
+1. Create a directory to use for the download location and change into that directory. For example `mkdir $HOME/dotnet_install && cd $HOME/dotnet_install`
+2. Run `curl -L https://aka.ms/install-dotnet-preview -o install-dotnet-preview.sh`
+3. Run the script with `sudo bash install-dotnet-preview.sh`
+
+Here's what the script does.
+
+* Detects the distribution and version. The script supports platforms and versions listed in [.NET 6.0 - Supported OS versions](https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0-supported-os.md).
+* Determines if additional system dependencies or utilities are needed to successfully complete and install them. For example `tar` is used to unpack that installer packages.
```

Comment:
I feel like the first sentence leaves the reader hanging.  What does it do with the information?  Does it install the dependencies or does it fail and notify me of what the dependencies are?

Reply:
This whole doc likely needs to be re-written. I just copied it from the release directory (like 5.0.4 or 6.0 preview 2). I just we merge and then I'll tackle fixing this doc.

Reply:
If you're trying to run on an unsupported distro, it tells you and stops the install attempt. During the dependency section, it checks to see if the components are installed and tells you if they're found. If not, there is feedback on what is being installed.